package.json

```diff
@@ -75,9 +75,9 @@
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
     "chai": "^3.5.0",
-    "chai-enzyme": "^0.5.2",
+    "chai-enzyme": "^0.8.0",
     "cross-env": "^3.1.4",
-    "enzyme": "^2.6.0",
+    "enzyme": "^3.0.0",
```

Comment:
You are missing `enzyme-adapter-react-16` too https://github.com/airbnb/enzyme#upgrading-from-enzyme-2x-or-react--16 :)

Reply:
Waiting for the chai-enzyme update.